src/domains/adapter/factory/adapterProviderImpl.ts

```diff
@@ -0,0 +1,17 @@
+import { type AdapterProvider } from '@domains/adapter/factory/adapterProvider';
+import { type ProjectManagementToolAdapter } from '../projectManagementToolAdapter';
+import { LinearAdapter } from '@domains/adapter/linear/linear.adapter';
+import { LinearDataRetriever } from '@domains/retriever/linear/linear.dataRetriever';
+
+const linearRetriever = new LinearDataRetriever();
+const linearAdapter = new LinearAdapter(linearRetriever);
+
+export class AdapterProviderImpl implements AdapterProvider {
+  private readonly adapters: ProjectManagementToolAdapter[] = [linearAdapter];
+
```

Comment:
Please add documentation to the method. You could use ConflictException too if you want to get the error map by the error handler.